things/service.go

```diff
@@ -36,6 +40,12 @@ var (
 
 	// ErrDisconnect indicates error in removing connection
 	ErrDisconnect = errors.New("remove connection failed")
+
+	// ErrCreateGroup indicates error in creating group.
+	ErrCreateGroup = errors.New("failed to create group")
+
+	// ErrFailedToRetrieveThings failed to retrieve things.
+	ErrFailedToRetrieveThings = errors.New("failed to retrieve things for group")
```

Comment:
Maybe better `failed to retrieve group things` or `failed to retrieve group members`